src/main/public/stylesheets/application.scss

```diff
@@ -22,7 +22,7 @@ $path: '/img/lib/';
 @import 'components/report-problem';
 @import 'components/dashboard';
 @import 'components/survey-links';
-@import 'components/mobile';
+@import 'components/phone';
```

Comment:
this shouldn't be changing as the file relates to how things are displayed on mobiles rather than asking about a phone number